www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/location.xml

```diff
@@ -243,6 +243,13 @@
     <advanced>true</advanced>
     <help>If you enable the WebSocket Support option, nginx will pass the upgrade header to the backed server.</help>
   </field>
+  <field>
+    <id>location.upstream_keepalive</id>
+    <label>Upstream Keepalive Support</label>
+    <type>checkbox</type>
+    <advanced>true</advanced>
+    <help>Adds the directives required for upstream keepalive to work (enables HTTP 1.1 and clears the Connection header). Keepalive parameters must be set in Upstream settings also.</help>
```

Comment:
If you clear the Connection header, you will break websockets. So This shuld be mentioned.
What can you do with that, which cannot be handled by enabling the checkbox for websocket support?

Reply:
added a warning about mutual exclusion to the help text (also validated in the model). "Connection" clearing is not for websockets breaking but needed for keepalives to work (so that the connection does not close).